### ISSUE TITLE
server crashed in version 1.9.5

### ISSUE THREAD

**Issue by tyan-boot** [Alcatergit/Hbm-s-Nuclear-Tech-GIT#116]
d0bafd2c18fb7054f8cc1347d0ec3667a19f85ed this commit add a new `FFUtils.getColorFromFluid` method which call `net.minecraft.client.Minecraft`，and in MainRegistry: https://github.com/Alcatergit/Hbm-s-Nuclear-Tech-GIT/blob/4c63c6054681b3fc63802abcd06b61abbc0b625f/src/main/java/com/hbm/main/MainRegistry.java#L1090

This will crash the server because `net.minecraft.client.Minecraft` doesn't exist in server, since the new `fluidColors` map is only for client render, may be we could move `registerFluidColors` to another place or add a check like 
```java
if (event.getSide().isClient()) {
    ModForgeFluids.registerFluidColors();
}
```

**Comment by slipwolff** [comment]
Can confirm this fixes issue #115 I reported earlier.

**Comment by Alcatergit** [comment]
Already fixed yesterday in dev. Fix will come out today.
